### PR TITLE
Added installation instructions to Readme

### PR DESCRIPTION
**Pull request by makermelissa**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -39,6 +39,31 @@ CircuitPython image, they have no value on a device already running CircuitPytho
...
 
+Installing from PyPI
+=====================
+
+On supported GNU/Linux systems like the Raspberry Pi, you can install the driver locally `from
+PyPI <https://pypi.org/project/adafruit-blinka-displayio/>`_. To install for current user:
+
+.. code-block:: shell
+
+    pip3 install Adafruit-Blinka
```
Comment (kattni) [inline README.rst:50]: Should this be capitalised?

File: README.rst [hunk 2]
```diff
@@ -39,6 +39,31 @@ CircuitPython image, they have no value on a device already running CircuitPytho
 The test suites in the test/src folder under **testing.universal** are by design
 intended to run on *either* CircuitPython *or* Micropython+compatibility layer to prove conformance.
 
+Installing from PyPI
+=====================
+
+On supported GNU/Linux systems like the Raspberry Pi, you can install the driver locally `from
+PyPI <https://pypi.org/project/adafruit-blinka-displayio/>`_. To install for current user:
```
Comment (kattni) [inline README.rst:46]: This link is incorrect, I believe.

File: README.rst [hunk 3]
```diff
@@ -39,6 +39,31 @@ CircuitPython image, they have no value on a device already running CircuitPytho
...
+
+.. code-block:: shell
+
+    pip3 install Adafruit-Blinka
+
+To install system-wide (this may be required in some cases):
+
+.. code-block:: shell
+
+    sudo pip3 install Adafruit-Blinka
```
Comment (kattni) [inline README.rst:56]: Same.

File: README.rst [hunk 4]
```diff
@@ -39,6 +39,31 @@ CircuitPython image, they have no value on a device already running CircuitPytho
...
+
+To install in a virtual environment in your current project:
+
+.. code-block:: shell
+
+    mkdir project-name && cd project-name
+    python3 -m venv .env
+    source .env/bin/activate
+    pip3 install Adafruit-Blinka
```
Comment (kattni) [inline README.rst:65]: Same.

File: README.rst (hunk 1 above)
Comment (makermelissa) [reply README.rst:50]: yes

File: README.rst (hunk 2 above)
Comment (makermelissa) [reply README.rst:46]: Oops, good catch.

File: README.rst (hunk 3 above)
Comment (makermelissa) [reply README.rst:56]: yes

File: README.rst (hunk 4 above)
Comment (makermelissa) [reply README.rst:65]: yes

**Comment by makermelissa** [comment]
I have it capitalized as it is on https://pypi.org/project/Adafruit-Blinka/, but it's case insensitive, so it really doesn't matter.

**Comment by makermelissa** [comment]
Thanks @kattni

**Pull request merged**